### PR TITLE
Remove all dead legacy code

### PR DESCRIPTION
**Pull request by jmauro**
Hello,

Since those template are not used anymore, we could remove those, don't you think?

cc @martinisoft or @onlyhavecans 

Regards,
JM

### CODE REVIEW

File: templates/default/exabgp.conf.erb [hunk 1]
```diff
@@ -1,10 +0,0 @@
-neighbor 127.0.0.1 {
```
Comment (martinisoft) [inline templates/default/exabgp.conf.erb]: This template is still required as a default: https://github.com/dnsimple/chef-exabgp/blob/5ab6b6fc03deeab874273ea99488fe890e7d569b/resources/exabgp.rb#L34

Comment (martinisoft) [reply templates/default/exabgp.conf.erb]: The fact you removed it without a test breaking reveals a regression we should be covering in our test cases.

**Review by martinisoft — CHANGES_REQUESTED** [review]
I can agree on the majority of these changes, but the template for the config needs to stay for sure. I'd like to make a failing test case for this too.

**Comment by jmauro** [comment]
Hello @martinisoft, I leave the default template alone and add a default cookbook source for the custom resource.

Let me know if this does the job

Regards,
JM

**Review by onlyhavecans — APPROVED** [review]

**Review by martinisoft — APPROVED** [review]

**Pull request merged**